features/FEATURE_BLE/ble/BLE.h

```diff
@@ -81,10 +81,10 @@ class BLEInstanceBase;
  * #include "ble/BLE.h"
  *
  * // declare the event queue, which the whole application will share.
- * static EventQueue event_queue( 4 * EVENTS_EVENT_SIZE);
+ * static EventQueue event_queue(4 * EVENTS_EVENT_SIZE);
  *
  * // Function invoked when there is a BLE event available.
- * // It put into the event queue the processing of the event(s)
+ * // The processing of the event(s) is then put into the event queue
```

Comment:
Event processing is put into the event queue.

Reply:
Done